### PR TITLE
SEO-174657-DotNET-MAUI-Busy-Indicator-Interlinking

### PR DESCRIPTION
**Pull request by Omanyadickson**
|Title|Description|
|--|--|
|Task Category| .NET MAUI Busy Indicator
|Content Task Link/Mail Screenshot | imagehttps://dev.azure.com/Syncfusion-General/Content/_workitems/edit/764224/ |
|UX task|NA|
|interlinking | PR - https://dev.azure.com/Syncfusion-General/Content/_workitems/edit/764746/|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/180203/|
|Excel/SharePoint link | https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI---Money-Keywords.aspx|
 
| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|.net maui busy indicator|0-10 |5FT|To move FT to the first position |Changed the content alignment, improved some content|
|.net maui busy indicator example|0-10 |2UG |To improve the FT position|added in the md file |
|.net maui busy indicator control|0-10 |4FT |To move FT to the first position |Demo link added |
|.net maui busy indicator animation |0-10 |1UG |To move FT to the first position | need to add the keyword in the FAQ section|
|.net maui busy indicator customization|0-10 |1FT| To move FT to the first position | added new section|
 
 
Worked on 16 keywords in the .NET MAUI Badge View and am keeping track of all keyword details in the link below:-
https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/Keyword-Plan-for-.NET-MAUI-Busy.aspx

### CODE REVIEW

File: MAUI/Busy-Indicator/Overview.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # .NET MAUI Busy Indicator (SfBusyIndicator) Overview
 
-The Busy Indicator control for .NET MAUI provides an indication of the app loading, data processing etc. It can be customized in terms of Indicator size, color, speed and more.
+The [Busy Indicator](https://www.syncfusion.com/maui-controls/maui-busy-indicator) control for .NET MAUI provides an indication of the app loading, data processing etc. It can be customized in terms of Indicator size, color, speed and more.
```
Comment (Sharly-Atieno) [inline MAUI/Busy-Indicator/Overview.md:13]: Revert this..

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 2]
```diff
@@ -161,5 +161,5 @@ The following gif image illustrates the result of the above code.
 
 ![Getting Started](Images/GettingStarted/CircularMaterial.gif)
 
-You can find the complete getting started sample of .NET MAUI Busy Indicator from this [link.](https://github.com/SyncfusionExamples/maui-busyindicator-samples)
+N> You can refer to our [.NET MAUI Busy Indicator](https://www.syncfusion.com/maui-controls/maui-busy-indicator) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Busy Indicator Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/BusyIndicator/SampleBrowser.Maui.BusyIndicator/Samples/BusyIndicator) that shows you how to render the BusyIndicator in .NET MAUI.
```
Comment (Sharly-Atieno) [inline MAUI/Busy-Indicator/Getting-Started.md:164]: Busy Indicator in .NET MAUI.
image

File: MAUI/Busy-Indicator/Overview.md (hunk 1 above)
Comment (Omanyadickson) [reply MAUI/Busy-Indicator/Overview.md]: Reverted

File: MAUI/Busy-Indicator/Getting-Started.md (hunk 2 above)
Comment (Omanyadickson) [reply MAUI/Busy-Indicator/Getting-Started.md]: The issue is resolved.

File: MAUI/Busy-Indicator/Overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About .NET MAUI Busy Indicator control | Syncfusion
+title: About .NET MAUI Busy Indicator Control | Syncfusion
 description: Learn here all about introduction of Syncfusion .NET MAUI Busy Indicator (SfBusyIndicator) control, its elements and more.
 platform: maui
 control: BusyIndicator
```
Comment (Sharly-Atieno) [inline MAUI/Busy-Indicator/Overview.md:6]: Revert this changes

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 4]
```diff
@@ -161,5 +161,5 @@ The following gif image illustrates the result of the above code.
 
 ![Getting Started](Images/GettingStarted/CircularMaterial.gif)
 
-You can find the complete getting started sample of .NET MAUI Busy Indicator from this [link.](https://github.com/SyncfusionExamples/maui-busyindicator-samples)
```
Comment (Sharly-Atieno) [inline MAUI/Busy-Indicator/Getting-Started.md:164]: Don't delete the above content

File: MAUI/Busy-Indicator/Overview.md (hunk 3 above)
Comment (Omanyadickson) [reply MAUI/Busy-Indicator/Overview.md:6]: Change Reverted as possible.

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 5]
```diff
@@ -161,5 +161,5 @@ The following gif image illustrates the result of the above code.
...
+N> You can refer to our [.NET MAUI Busy Indicator](https://www.syncfusion.com/maui-controls/maui-busy-indicator) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Busy Indicator Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/BusyIndicator/SampleBrowser.Maui.BusyIndicator/Samples/BusyIndicator) that shows you how to render the Busy Indicator in .NET MAUI.
```
[the remark this answers is not in the document]
Comment (Omanyadickson) [reply MAUI/Busy-Indicator/Getting-Started.md:164]: Resolved as possible.

File: MAUI/Busy-Indicator/Getting-Started.md (hunk 4 above)
Comment (Omanyadickson) [reply MAUI/Busy-Indicator/Getting-Started.md:164]: The content has been retrieved.
